### PR TITLE
Roulette guide

### PR DESCRIPTION
**Pull request by vanshwassan**
- Added the roulette guide under the QRNG Guides section.
- Covers the entire roulette smart contract + steps on how to deploy and interact with it using Remix IDE.

### CODE REVIEW

**Review by Ashar2shahid — CHANGES_REQUESTED** [review]
it would be helpful if the coding section can be divided into sections to navigate on the right hand side bar

File: docs/guides/qrng/roulette-guide/index.md [hunk 1]
```diff
@@ -0,0 +1,871 @@
...
+Number
+EvenOdd
+Third
+Half
+```
+
+These values represent different types of bets that players can make in the game
+of Roulette.
+
+```solidity
```
[the remark this answers is not in the document]
Comment (vanshwassan) [reply docs/guides/qrng/roulette-guide/index.md]: Done 👍🏻

**Comment by vanshwassan** [comment]
> it would be helpful if the coding section can be divided into sections to navigate on the right hand side bar

Yeah, I've added some sub-headings. LMK if it looks good to you.

**Review by Ashar2shahid — APPROVED** [review]

**Comment by Ashar2shahid** [comment]
@wkande when you get the time can you review this ?

**Comment by wkande** [comment]
@vanshwassan @Ashar2shahid 

1.  I would add a period after the sentence: "The complete contract code can be found here"
2.  And here as well: "Fill in the amount in wei under VALUE on the tab above and click on topUpSponsorWallet()"
3.  Again: "use betEvenOdd() to bet on either the set of all even or odd numbers on the board"

Other than that this looks good. Of course I did not actually create the app but I feel it will work.

**Comment by wkande** [comment]
@vanshwassan 

For links that take the reader to an external site add the component <ExternalLinkImage/> to the label.

`[here](https://github.com/Ashar2shahid/qrng-roulette/blob/main/contracts/contracts/Roulette.sol)`

`[here<ExternalLinkImage/>](https://github.com/Ashar2shahid/qrng-roulette/blob/main/contracts/contracts/Roulette.sol)`

Not that there is no space between the label and the component.

**Review by Ashar2shahid — APPROVED** [review]

**Pull request merged**